app/lib/actions/katello/host/register.rb

```diff
@@ -108,8 +108,9 @@ def plan_subscription_facet(host, activation_keys, consumer_params)
           subscription_facet = host.subscription_facet || ::Katello::Host::SubscriptionFacet.new
           subscription_facet.host = host
           subscription_facet.update_from_consumer_attributes(consumer_params)
-          subscription_facet.save!
+          subscription_facet.save! # persist to obtain an id
```

Comment:
@ehelms I tried that just now and I still see the failures. Will continue to brainstorm.